linkerd/app/outbound/src/zone.rs

```diff
@@ -0,0 +1,79 @@
+use crate::metrics::prom;
+use linkerd_app_core::{
+    metrics::OutboundZoneLocality, svc, transport::metrics::zone::TcpZoneMetrics,
+};
+use prometheus_client::{
+    encoding::{EncodeLabelSet, EncodeLabelValue},
+    registry::Unit,
+};
+
+#[derive(Clone, Debug, Default)]
+pub struct TcpZoneMetricsParams {
+    transfer_cost: prom::Family<TcpZoneLabels, prom::Counter>,
+}
+
+impl TcpZoneMetricsParams {
+    pub fn register(registry: &mut prom::Registry) -> Self {
+        let transfer_cost = prom::Family::default();
+        registry.register_with_unit(
+            "transfer_cost",
+            "The total cost of data transferred in and out of this proxy in bytes",
```

Comment:
More yakshaving, not a blocker:

```suggestion
            "The total amount data transferred in and out of this proxy, by cost zone",
```